src/ApiCompat/Microsoft.DotNet.ApiCompat.Task/ValidatePackageTask.cs

```diff
@@ -34,11 +34,6 @@ public class ValidatePackageTask : TaskBase
         /// </summary>
         public string? RuntimeGraph { get; set; }
 
-        /// <summary>
-        /// A NoWarn string contains the error codes that should be ignored.
-        /// </summary>
-        public string? NoWarn { get; set; }
-
```

Comment:
Unrelated change. Just moved the NoWarn down to mimic the other task and the CLI order.